smart_campus/app/views.py

```diff
@@ -113,14 +119,145 @@ def index(request):
 
 @login_required
 def station_list_page(request):
-    context = {'email': request.user.email}
+    """Show all stations managed by the user's group"""
+    if request.user.can(Permission.ADMIN):
+        stations = Station.objects.all()
+    else:
+        stations = Station.objects.filter(owner_group=request.user.group)
+
+    station_data = [
+        {
+            'id': station.id,
+            'name': station.name,
+            'image_url': (
+                '' if not station.stationimage_set.exists()
+                else
+                '/{0}'.format(station.stationimage_set.get(is_primary=True).image.url)
```

Comment:
Is it possible to write an `image_url` property in Django model?

Reply:
yep
I'll add one in station, saving the url for the primary image